src/Table/Column/Link.php

```diff
@@ -55,7 +55,7 @@ class Link extends Table\Column
      */
     public $args = [];
 
-    /** @var bool use value as label of the link */
+    /** @var bool|string use value as label of the link */
     public $use_label = true;
```

Comment:
no string for "use_xxx"

Reply:
I would normally propose using $label.
$label = false - no label
$label = true - default label template
$label = 'string' - use it as template

_(It works pretty nice in the new filestore Upload Control having such feature)_

Reply:
yes